package.json

```diff
@@ -144,7 +144,7 @@
     "jest-watch-typeahead": "0.4.2",
     "lerna": "3.20.2",
     "lerna-changelog": "1.0.0",
-    "lint-staged": "9.5.0",
+    "lint-staged": "10.0.0",
```

Comment:
We need to update the config file as well

Reply:
True. Done in 18b. Thanks for checking!